### ISSUE TITLE
LAshow -o dissapeared

### ISSUE THREAD

**Issue by ebioman** [thegenemyers/DALIGNER#16]
Hello
I was just curious why the option -o "optimal" for LAshow disappeared . I liked the idea behind it , used it in a few scripts and am now curious whether the concept did not hold up or whether it was a technical decisions ?
Cheers

**Comment by thegenemyers** [comment]
-o was for "overlaps only" not "optimal".  All pacbio alignments are 
pretty much "optimal" in
that the minimize the edit distance between the two aligned sections.  
The idea of the -o option
was to show only local alignments that are end-to-end overlaps.  I have 
not found the subclass
of these alignment useful given that Pacbio reads rarely so overlap.  If 
you still think the option
as I've described has merit, I could add back the option in a future 
release.  If you had another
idea, then if you explain it to me better, I might consider it.

    Cheers,
        Gene

P.S.  How is the software working out for you these days -- did you make 
it through
           your previous issues?

**Comment by pb-jchin** [comment]
@thegenemyers I did have some code for Falcon output only overlaps without overhangs. This is for constructing the string graph after overlapping the error corrected reads. I did not modify your code but create a new binary based on `LAShow`.

**Comment by ebioman** [comment]
Thank you Gene

Indeed I mixed up the terms, I meant overlaps. The function is not crucial but it was helpful for short regions with low coverage to get an impression about their spatial organization. To just generating simple overlap graphs to identify e.g. clusters of reads which might be interesting to analyze in more detail. 

Otherwise I am currently not too much using the DALIGNER  itself anymore, mostly for the previous described scenario and to analyze the average coverage on each read (helpful to identify potential “metagenomes” and repeats”). I do use Falcon with DALIGNER quite a bit lately which works mostly well for me. 

Cheers Emanuel











From: Eugene W Myers Jr
Sent: ‎Wednesday‎, ‎April‎ ‎22‎, ‎2015 ‎8‎:‎27‎ ‎PM
To: thegenemyers/DALIGNER
Cc: Emanuel Schmid




-o was for "overlaps only" not "optimal". All pacbio alignments are 
pretty much "optimal" in
that the minimize the edit distance between the two aligned sections. 
The idea of the -o option
was to show only local alignments that are end-to-end overlaps. I have 
not found the subclass
of these alignment useful given that Pacbio reads rarely so overlap. If 
you still think the option
as I've described has merit, I could add back the option in a future 
release. If you had another
idea, then if you explain it to me better, I might consider it.

Cheers,
Gene

P.S. How is the software working out for you these days -- did you make 
it through
your previous issues?

**Comment by ebioman** [comment]
Hello Jason

So is this binary part of Falcon? Then I will just use this instead.

Cheers

Emanuel











From: Jason Chin
Sent: ‎Wednesday‎, ‎April‎ ‎22‎, ‎2015 ‎8‎:‎30‎ ‎PM
To: thegenemyers/DALIGNER
Cc: Emanuel Schmid





@thegenemyers I did have some code for Falcon output only overlaps without overhangs. This is for constructing the string graph after overlapping the error corrected reads. I did not modify your code but create a new binary based on LAShow.
